MmSupervisorPkg/Core/MmSupervisorCore.inf

```diff
@@ -22,16 +22,16 @@
   BASE_NAME                      = MmSupervisorCore
   FILE_GUID                      = 4E4C89DC-A452-4B6B-B183-F16A2A223733
   MODULE_TYPE                    = MM_CORE_STANDALONE
-  VERSION_STRING                 = 6.003
+  VERSION_STRING                 = 7.001

```

Comment:
Just curios but what is the criteria for updating the major version of the core?  Just want to know why it's being update here versus other times I've seen the core updated.

Reply:
The version will be bumped up whenever it is ready to make a release :) This change update the major version because the MU foundation is updated to 202208. I meant to do that after 202208 support is in, but it caused some issues when building with GCC. These should fix those issues and make it ready enough for a release.